src/community/json-ld/src/main/resources/META-INF/services/org.opengis.filter.expression.Function

```diff
@@ -0,0 +1,2 @@
+org.geoserver.jsonld.expressions.FilterFunction_xpath
+org.geoserver.jsonld.expressions.FilterFunction_reproject
```

Comment:
FilterFunction_ is not required (comes from old code generator), recommend XPathFunction and ReprojectFunction

Reply:
fixed